src/main/java/edu/kit/ipd/crowdcontrol/objectservice/Main.java

```diff
@@ -196,15 +188,16 @@ private static void boot(DatabaseManager databaseManager, List<Platform> platfor
         DatabaseMaintainer maintainer = new DatabaseMaintainer(databaseManager.getContext(), cleanupInterval);
         maintainer.start();
 
-        MailFetcher mailFetcher = getMailFetcher(mailDisabled, sendMail);
-        MailSender mailSender = getMailSender(mailDisabled, sendMail);
+        MailFetcher mailFetcher = getMailFetcher(mailDisabled, getConfig().mail.moneyReceiver);
```

Comment:
mhmm other words for that is that the config is incomplete. Shouldnt we find some other word ?